android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/PlaintextSharedPreferencesDAO.java

```diff
@@ -0,0 +1,128 @@
+package org.sagebionetworks.bridge.android.manager;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.google.common.collect.Maps;
+import com.google.gson.reflect.TypeToken;
+
+import org.sagebionetworks.bridge.rest.RestUtils;
+import org.sagebionetworks.bridge.rest.model.ConsentSignature;
+import org.sagebionetworks.bridge.rest.model.SignIn;
+import org.sagebionetworks.bridge.rest.model.StudyParticipant;
+import org.sagebionetworks.bridge.rest.model.UserSessionInfo;
+
+import java.lang.reflect.Type;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentMap;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * FIXME: encrypt sensitive information, integrate with fingerprint
+ */
+
+public class PlaintextSharedPreferencesDAO implements AccountDAO, ConsentDAO {
```

Comment:
I'm not sure you did the class hierarchy of this right.  I'd suggest switching AccountDao and ConsentDao to be there own classes that extend PlaintextSharedPreferencesDao, where that is the class that has the functionality.